cmd/prometheus/main.go

```diff
@@ -205,6 +205,9 @@ func main() {
 	a.Flag("query.max-samples", "Maximum number of samples a single query can load into memory. Note that queries will fail if they would load more samples than this into memory, so this also limits the number of samples a query can return.").
 		Default("50000000").IntVar(&cfg.queryMaxSamples)
 
+	a.Flag("web.cors.origin", "CORS origin to use").
+		PlaceHolder("<URL>").StringsVar(&cfg.web.CORSOrigins)
```

Comment:
Hmm, I wonder if this would be best as a regex.

Reply:
@brian-brazil 
regex as in like this?:
`--web.cors.origin='^(http|https)://(domain1|domain2).(com)$'`

Reply:
Yes, though we anchor regexes for users